modules/secure-cd/outputs.tf

```diff
@@ -23,3 +23,8 @@ output "binauthz_policy_required_attestations" {
   description = "Binary Authorization policy required attestation in GKE projects"
   value       = [for policy in google_binary_authorization_policy.deployment_policy : policy.cluster_admission_rules.*.require_attestations_by]
 }
+
+output "delivery_pipeline_name" {
```

Comment:
Name is set by user. Lets output a computed attrib like id instead
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/clouddeploy_delivery_pipeline#id